v3/mathjax/legacy/jax/element/MmlNode.js

```diff
@@ -0,0 +1,110 @@
+(function () {
+  var MML = MathJax.ElementJax.mml;
+  
+  var PROPERTY = [
+    'movesupsub',
+    'subsupOK',
+    'primes',
+    'movablelimits',
+    'scriptlevel',
+    'open',
+    'close',
+    'isError',
+    'multiline',
+    'variantForm',
+    'autoOP'
+  ];
+
+  MML.mbase.Augment({
+    toMmlNode: function (factory) {
+      var m = this.data.length;
+      var kind = this.type; if (kind === 'texatom') kind = 'TeXAtom';
```

Comment:
This is a legacy file that is only used as glue to get from the old internal format to the new internal format, so this is not going to end up as part of v3.  So I'm not too worried about format, here.